iptables/parameters/match_owner.go

```diff
@@ -47,12 +49,25 @@ func uid(id string, negative bool) *OwnerParameter {
 	}
 }
 
+func uidRangeOrValue(uIDsToPorts config.UIDsToPorts) *OwnerParameter {
+	return &OwnerParameter{
+		flag:  "--uid-owner",
+		value: string(uIDsToPorts.UIDs),
+	}
+}
+
 // Uid matches if the packet socket's file structure (if it has one) is owned by the user
 // with given UID
 func Uid(id string) *OwnerParameter {
 	return uid(id, false)
 }
 
+// UidRangeOrValue matches if the packet socket's file structure (if it has one) is owned by the user
+// with given list of UID values or ranges
+func UidRangeOrValue(uIDsToPorts config.UIDsToPorts) *OwnerParameter {
+	return uidRangeOrValue(uIDsToPorts)
+}
+
```

Comment:
I think we don't have to create the local function if we are not reusing it anywhere. Wdyt?

```suggestion
// Uid matches if the packet socket's file structure (if it has one) is owned by the user
// with given UID
func Uid(id string) *OwnerParameter {
	return uid(id, false)
}

// UidRangeOrValue matches if the packet socket's file structure (if it has one) is owned by the user
// with given list of UID values or ranges
func UidRangeOrValue(uIDsToPorts config.UIDsToPorts) *OwnerParameter {
	return &OwnerParameter{
		flag:  "--uid-owner",
		value: string(uIDsToPorts.UIDs),
	}
}

```